bundles/pubsub/test/test/serializer_activator.cc

```diff
@@ -0,0 +1,87 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#include "celix_constants.h"
+#include <unistd.h>
+
+#include "celix_api.h"
+#include "../../pubsub_spi/include/pubsub_message_serialization_service.h"
+#include "../../pubsub_spi/include/pubsub_constants.h"
+#include "msg.h"
+
+struct activator {
+    long serId;
+    pubsub_message_serialization_service_t svc;
+};
+
+celix_status_t bnd_start(struct activator *act, celix_bundle_context_t *ctx) {
+
+#pragma GCC diagnostic ignored "-Wunused-parameter"
```

Comment:
is this needed or can be parameters with names (c++) be used?

Reply:
This is needed because the lambdas ignore parameters. Even when simply removing the names, I kept getting errors. Since this is just a test file, I opted for ignoring the warnings (which get turned into errors due to `-Werrors`).